src-docs/src/components/codesandbox/link.js

```diff
@@ -108,6 +106,12 @@ export const Demo = () => (<EuiButton>Hello world!</EuiButton>);
     );
   }
 
+  // Prepend the `@emotion/react` JSX pragma to enable the `css` prop on non-EUI elements/components
+  // https://emotion.sh/docs/css-prop#jsx-pragma
+  demoContent = `/** @jsxImportSource @emotion/react */
+${demoContent}
+`;
```

Comment:
Very extra, but I don't suppose we could detect a `css=` prop and only prepending this pragma if it exists? I just worry about potentially confusing devs or people blindly copy/pasting extra boilerplate cruft that they don't need 🙈 

Something like
```suggestion
  if (demoContent.includes('css={')) {
    demoContent = `/** @jsxImportSource @emotion/react */

${demoContent}
`;
  }
```

Reply:
Limiting copy-paste errors is always a good idea. Will update